src/components/Modals/AnnouncementModal.vue

```diff
@@ -6,7 +6,14 @@
           <div class="announce-title">{{announcement.title}}</div>
           <div class="announce-date">{{ createdString }}</div>
         </div>
-        <div class="announce-body">{{announcement.description}}</div>
+        <div class="announce-body">
+          {{announcement.description}}
+          <access-control :roles="[USER[ROLE.ADMIN]]">
+            <br/>
```

Comment:
Why do we need the `<br />`? We should avoid using those where possible

Reply:
It appears this was just adding space. I took it out and added some padding with CSS instead.